modules/n1ql/examples/dml/delete-offset.jsonc

```diff
@@ -0,0 +1,12 @@
+id
```

Comment:
A bit of a nit - the "Results" do not match the the results returned by its corresponding example query. There are no "ids" returned by the corresponding DELETE query.

Reply:
that's right, per my question above to Simon @dhanyagowrish I did not include the SELECT id FROM... statements that I used here and above, not sure how detailed these examples tend to be, happy to add/reformat if that is the standard

Reply:
I reworded to clarify that this is the Table view, not JSON. 2x